### PR TITLE
Update common.less

### LINKED ISSUE #1009: Remove iframe borders on Vimeo embed
Vimeo embed has border around `iframe`. It should be removed.
https://busy.org/design/@design-guy/graffiti-house-branding

### PR DESCRIPTION
**Pull request by Kennybll**
Fixes #1009.
Changes:
- Removes border around iframes.
- Adds a 20px margin on the top and bottom to create a unanimous feeling with images.

### CODE REVIEW

File: src/styles/common.less [hunk 1]
```diff
@@ -49,3 +49,9 @@ video {
 .iconfont {
   font-size: 26px;
 }
+
+iframe {
+  border: none;
+  margin-top: 20px;
```
Comment (jm90m) [inline src/styles/common.less]: I'm not sure if we should add these margins in, the author of the post might not have intended there to be margins around their iframe.

Comment (jm90m) [reply src/styles/common.less]: @bonustrack what do you think?

Comment (bonustrack) [reply src/styles/common.less]: The margin create an extra space in feed:
image
This PR is good for solve the issue about iframe border, for the margin we can do it in another PR, we should use a specific style class for it.

Comment (jm90m) [reply src/styles/common.less]: @Kennybll Did you want to solve the issue for the margins, what @bonustrack talked about above in this PR or in another PR? If you want to do it in another PR, please close this one :)

**Pull request closed without merging**